blockcypher/api.py

```diff
@@ -1624,7 +1624,7 @@ def broadcast_signed_transaction(unsigned_tx, signatures, pubkeys, coin_symbol='
 
 
 def simple_spend(from_privkey, to_address, to_satoshis, change_address=None,
-        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc'):
+        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc', preference: str = 'low'):
```

Comment:
For a better UX I suggest that keep the fee `high` here.

Reply:
Thanks for your feedback @quentinlesceller. I plan making the needed changes. Also noticed a similar need for the function `simple_spend_p2sh`. Will attempt to make similar changes and add tests as well.

Thanks